libs/vkd3d/state.c

```diff
@@ -4479,13 +4687,13 @@ static HRESULT d3d12_pipeline_state_init_graphics_create_info(struct d3d12_pipel
                 break;
 
             case VK_SHADER_STAGE_FRAGMENT_BIT:
-                if ((ret = vkd3d_shader_parse_output_signature(&dxbc, &output_signature)) < 0)
+                if ((ret = vkd3d_shader_parse_output_signature(&dxbc, &ps_output_signature)) < 0)
                 {
                     hr = hresult_from_vkd3d_result(ret);
                     goto fail;
                 }
 
-                if (FAILED(hr = d3d12_pipeline_state_validate_blend_state(state, device, desc, &output_signature)))
+                if (FAILED(hr = d3d12_pipeline_state_validate_blend_state(state, device, desc, &ps_output_signature)))
```

Comment:
Couldn't this reuse io_output_signature?

Reply:
I guess in this case we can since the PS output signature is only used for blend state validation.